app/assets/javascripts/discourse/app/models/post-stream.js

```diff
@@ -15,6 +15,19 @@ import { isEmpty } from "@ember/utils";
 import { loadTopicView } from "discourse/models/topic";
 import { schedule } from "@ember/runloop";
 
+let _lastEditNotificationClick = null;
```

Comment:
This seems like a fine solution, however we should add a `resetLastEditNotification()` method (or similar name) that will set it back to `null`. Then add it to the test suite between tests to avoid leaking state.

Reply:
Done